.pre-commit-config.yaml

```diff
@@ -31,7 +31,7 @@ repos:
     - id: check-hooks-apply
 
 - repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v4.6.0
+  rev: v5.0.0
```

Comment:
We should probably start using the new `check-illegal-windows-names` and maybe the fixed `requirements-txt-fixer`

Reply:
`requirements-txt-fixer` did damage in #17305, so we should only add `check-illegal-windows-names`